app/models/rpc/package.scala

```diff
@@ -146,6 +147,11 @@ package object rpc {
   implicit val StreamStateReads = enumReads(StreamState)
   implicit val DSAResponseFormat: Format[DSAResponse] = Json.format[DSAResponse]
 
+  implicit val QoSFormat: Format[models.akka.QoS.Level] = new Format[QoS.Level] {
```

Comment:
Since you import `models.akka.QoS`, you don't need a fully qualified class name here.

Reply:
removed all this formatters since there is no need to serialize it (some changes in stream topology)